### ISSUE TITLE
Feature request: Activate UI gesture zoom in maps

### ISSUE THREAD

**Issue by LarsSjogreen** [akvo/akvo-lumen#1445]
# Context
Some users have pointed out that it would be convenient to be able to zoom the maps using the built in zoom gestures of the operating system (usually pinch-to-zoom and/or mouse scroll wheel).

# Solution or next step
There probably is (at least) one challenge with this feature request and that is that scroll wheel scrolls the web page and that any map under the mouse pointer will "highjack" the scrolling making for a very bad user experience. There are ways and (I think) some best practices available to distinguish between scrolling behaviour and zooming behaviour. For finger gestures the single finger panning function now seems to be phazed out from more and more web pages in favor of the combined two finger pan/zoom gesture (together with a textual UI hint pointing this out to the user). I imagine that some CMD/Ctrl + [scroll wheel] gesture could well be used for the zoom function in Lumen so to not interfere with page scrolling.

In short: having some kind of natural feeling zoom function in the Lumen maps would be nice. (At least from what I think. Using the +/- buttons for zooming is quite annoying.)
